CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
-## 1.10.0-beta.1
+## 1.10.0
 
 ### ✨ Features
-* Add `mapboxgl.preloadWorkerPool()` and `mapboxgl.removePreloadedWorkerPool()` methods to allow developers to optimize load times for their maps ([#9391](https://github.com/mapbox/mapbox-gl-js/pull/9391))
+* Add `mapboxgl.prewarm()` and `mapboxgl.clearPrewarmedResources()` methods to allow developers to optimize load times for their maps ([#9391](https://github.com/mapbox/mapbox-gl-js/pull/9391))
```

Comment:
Fixed incorrect names here, as they changed while the PR was finalized and we missed it from the changelog.